subsys/shell/shell_cmds.c

```diff
@@ -40,6 +40,12 @@
 #define SHELL_HELP_ECHO_OFF	\
 	"Disable shell echo. Editing keys and meta-keys are not handled"
 
+#define SHELL_HELP_SELECT	"Selects new root command. In order for the " \
+	"command to be selected, it must meet the criteria:\n"		      \
+	" - it is a static command\n"					      \
+	" - it is not preceded by a dynamic command\n"			      \
+	"Return to the main command tree is done by pressing ctrl+s."
```

Comment:
Ctrl+S/Ctr+Q is already used to send XOFF/XON characters over serial line.

Reply:
fixed